### PR TITLE
Issue #2456 Fixed the typo in start_preflight.go.

### LINKED ISSUE #2456: Typo in preflight checks
File preflight.go contains a typo: https://github.com/minishift/minishift/blob/1426bddcbf7aae0b8fab29dbe28b667a9ae218cb/cmd/minishift/cmd/start_preflight.go#L62

### Expected

```
-- Check if deprecated options are used ... OK
```

### Actual

```
-- Check if depereccated options are used ... OK
```

### PR DESCRIPTION
**Pull request by ymdatta**
Fix #2456

### CODE REVIEW

**Comment by budhrg** [comment]
add to whitelist

**Comment by praveenkumar** [comment]
@ymdatta looks like your commit message is not as expected. There shouldn't be any `.` between `Issue #2456` and `Fixed`

```
 Issue #2456 Fixed the typo in start_preflight.go.
```

**Comment by ymdatta** [comment]
@budhrg , fixed the commit message.

File: cmd/minishift/cmd/start_preflight.go [hunk 1]
```diff
@@ -59,7 +59,7 @@ func preflightChecksBeforeStartingHost() {
 	preflightCheckSucceedsOrFails(
 		configCmd.SkipDeprecationCheck.Name,
 		checkDeprecation,
-		"Check if depereccated options are used",
+		"Check if deprecated options are used",
```
Comment (praveenkumar) [inline cmd/minishift/cmd/start_preflight.go]: LGTM 👍

**Comment by budhrg** [comment]
Thanks for the contribution @ymdatta  :+1:

**Comment by ymdatta** [comment]
@budhrg , @praveenkumar : Thanks for all the help :)

**Pull request merged**